ultrasonic/src/main/res/values-pt/strings.xml

```diff
@@ -334,7 +334,7 @@
     <string name="util.bytes_format.megabyte">0.00 MB</string>
     <string name="util.no_time">&#8212;:&#8212;&#8212;</string>
     <string name="util.zero_time">0:00</string>
-    <string name="video.get_mx_player_text">O localMediaPlayer MX não está instalado. Descarregue da graça pela Play Store ou modifique as configurações de vídeo.</string>
+    <string name="video.get_mx_player_text">O Player MX não está instalado. Descarregue da graça pela Play Store ou modifique as configurações de vídeo.</string>
```

Comment:
This should be "MX Player" instead of "Player MX"

Reply:
Yep those were strange for me too, but those were the original (maybe google translated) strings.
Anyway, they're fixed now.